<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>231</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>231</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>231</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="10,3">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_7">
     <property name="leftMargin">
      <number>250</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="reject">
       <property name="text">
        <string>취소</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="accept">
       <property name="text">
        <string>확인</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="2">
     <item row="1" column="0" rowspan="4">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;새 비밀번호 확인 : &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;기존 비밀번호 : &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;새 비밀번호 : &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QLineEdit" name="New_password">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLineEdit" name="Confirm_password">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <widget class="QLineEdit" name="Old_password">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
